.editorconfig

```diff
@@ -5,6 +5,24 @@ end_of_line = crlf
 charset = utf-8
 indent_style = space
 indent_size = 4
+dotnet_style_operator_placement_when_wrapping = beginning_of_line
+tab_width = 2
+dotnet_style_coalesce_expression = true:suggestion
+dotnet_style_null_propagation = true:suggestion
+dotnet_style_prefer_is_null_check_over_reference_equality_method = true:suggestion
+dotnet_style_prefer_auto_properties = true:silent
```

Comment:
Why `silent`, why not `suggestion`? For the others too.

Reply:
These are default values that VS defined when I opened .editorconfig, I didn't touch this file myself. I assumed the VS defaults would be ok. I can revert these changes but the next in line to open it will have the same delta.